src/logics/calculate.js

```diff
@@ -1,21 +1,16 @@
 import operate from './operate';
 
+const ops = ['+', '-', 'X', '/', '%'];
+
 const calculate = (calObj, btnName) => {
   const obj = calObj;
-  switch (btnName) {
-    case '+/-':
-      obj.total *= -1;
-      obj.next *= -1;
-      break;
-    case '+':
-    case '-':
-    case '*':
-    case '/':
-    case '%':
-      obj.total = operate(calObj.total, calObj.next, btnName);
-      break;
-    default:
+  if (ops.includes(obj.operation)) {
+    obj.total = operate(obj.total, obj.next, obj.operation);
+  } else {
+    obj.total *= -1;
```

Comment:
Think about the tweak you'll apply to fix a very little bug on the app's logic

When i open your calculator app, if i click on a number (say `"5"`), then i click on `"="`, i will get ` -5 ` at the display